src/main/java/eu/dissco/backend/repository/SpecimenRepository.java

```diff
@@ -22,40 +21,28 @@ public class SpecimenRepository {
   private final DSLContext context;
   private final ObjectMapper mapper;
 
-  public DigitalSpecimen getLatestSpecimenById(String id) {
-    return context.select(NEW_DIGITAL_SPECIMEN.asterisk())
-        .from(NEW_DIGITAL_SPECIMEN)
-        .where(NEW_DIGITAL_SPECIMEN.ID.eq(id))
+  public DigitalSpecimenWrapper getLatestSpecimenById(String id) {
+    return context.select(DIGITAL_SPECIMEN.asterisk())
+        .from(DIGITAL_SPECIMEN)
+        .where(DIGITAL_SPECIMEN.ID.eq(id))
         .fetchOne(this::mapToDigitalSpecimen);
```

Comment:
Don't you need to check if this is the latest version? Wouldn't fetchOne() fail if we have multiple versions of the same specimen?

Reply:
This is the db there is only one version on the specimen which is always the latest.